src/Elcodi/Component/Product/Entity/Interfaces/PurchasableInterface.php

```diff
@@ -40,6 +40,13 @@
     DimensionableInterface
 {
     /**
+     * Gets name
+     *
+     * @return string
+     */
+    public function getName();
```

Comment:
This interface is already defined inside `ProductInterface`, interface that extends this one. So this means that there is an architectural problem here.

@alch